docs/vault/guide.md

```diff
@@ -52,9 +52,15 @@ subkey generate --output-type json | jq '{"polkabtcvault": .secretPhrase}' > key
 
 ### 3. Start the Vault client
 
-You can run the entire Vault client and the Bitcoin node with the following command:
+?> If you already have a locally running Bitcoin testnet node, only start the vault client:
 
-```sh
+```shell
+docker-compose up vault
```

Comment:
Not sure if this will work because we link the container name

Reply:
For me, it works locally :)